src/components/icon/Icon.vue

```diff
@@ -25,17 +34,9 @@
              * internal icons are always MDI.
              */
             newIcon() {
-                if (!this.both) {
-                    if (this.newPack === 'mdi') {
-                        return `${this.newPack}-${this.icon}`
-                    } else {
-                        return `fa-${this.icon}`
-                    }
-                }
-
                 return this.newPack === 'mdi'
                     ? `${this.newPack}-${this.icon}`
-                    : `fa-${this.getEquivalentIconOf(this.icon)}`
+                    : this.addFAPrefix(this.getEquivalentIconOf(this.icon))
```

Comment:
Why are you always loading `getEquivalentIcon` ?

Reply:
I'm not sure I get what you mean. I only added `addFAPrefix`. The call to `getEquivalentIconOf` was there before.

Reply:
Ah I see, you are asking why I removed the section L28-L35.
I remember removing it because I could not figure out what this does. In fact, it caused problems with icons that are named differently in FA.

For example, the Message has the `both` prop set to true and uses the `information` icon. But this icon is named `info` in FA. So why is this not renamed to the correct icon-name?

Reply:
Because the project uses mdi name and for internal icons we load the equivalent.

Reply:
Exactly, but my point is that the equivalent icon is Not loaded when `both = true`. 
My change fixes this already.